api/container/containerv1/properties.go

```diff
@@ -0,0 +1,53 @@
+package containerv1
+
+import (
+	"fmt"
+
+	"github.com/IBM-Cloud/bluemix-go/client"
+)
+
+//Vlan ...
+type DCVlan struct {
+	ID         string           `json:"id"`
+	Properties DCVlanProperties `json:"properties"`
+	VlanType   string           `json:"type"`
+}
+
+//VlanProperties ...
+type DCVlanProperties struct {
+	LocalDiskStorageCapability string `json:"local_disk_storage_capability"`
+	Location                   string `json:"location"`
+	Name                       string `json:"name"`
+	Note                       string `json:"note"`
+	PrimaryRouter              string `json:"primary_router"`
+	SANStorageCapability       string `json:"san_storage_capability"`
+	VlanNumber                 string `json:"vlan_number"`
+	VlanType                   string `json:"vlan_type"`
+}
+
+//Subnets interface
+type Vlans interface {
+	List(string, ClusterTargetHeader) ([]DCVlan, error)
```

Comment:
With the latest API's the ClusterTargetHeader is not required and none of the API docs mention this. So please remove it